app/views/teaching_events/index.html.erb

```diff
@@ -5,27 +5,6 @@
     <p>
       Find out more about getting into teaching at a free event where you can get all your questions answered.
     </p>
-
```

Comment:
@philspray Sorry - I wasn't very clear on the Trello ticket about the accordion!

I think we probably still need some description about the two remaining event types - but because there are only two, we might be able to ditch the accordion and just have text on the page

That depends on how much space this takes up on mobile though, and whether that will stop people getting to the filters?